components/aside.js

```diff
@@ -0,0 +1,19 @@
+import { jsx } from "nano-jsx";
+import { Seperator } from "./seperator.js";
+
+export const Aside = ({ license, files }) => {
```

Comment:
Just a matter of preference but please use `function` instead of `const` to declare functional components.